### DISCUSSION TITLE
Where is the min/max temperature information coming from for eTRVs?

### DISCUSSION THREAD

**Discussion by mzurhorst** [SukramJ/aiohomematic#794] [Q&A] [answered]
Hi.

While looking into the mapping of the profiles, I now realize that Home Assistant shows we wrong values for "min_temp" and "max_temp".  They do not match what I have set up in Raspberrymatic.   
Where are those coming from? -- Would this be a bug?

Regards,
  Marcus

**Home Assistant**
![grafik](https://user-images.githubusercontent.com/575190/210049481-d4c2a572-3b0d-4278-bfc1-9c1f5321f66f.png)

**HomematicIP Device and Heat Group settings  (identical):**
![grafik](https://user-images.githubusercontent.com/575190/210049429-ef9b10e7-2d51-47a7-a176-73b5bf240f11.png)

**Comment by SukramJ** [comment, marked as answer]
The dynamically reading of the min/max has been added with [1.19.7](https://github.com/danielperna84/custom_homematic/releases/tag/1.19.7)
According to the release notes:
> Read min/max temperature for climate devices. To make this work you need to execute the service homematicip_local.clear_cache followed by a HA restart.

**Reply by mzurhorst** [reply]
Ok, got it.  I missed to clear the cache, my HA is starting frequently.
